plenary/plenary-24.yaml

```diff
@@ -0,0 +1,168 @@
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: It is critically important that all member organizations who have maintenance responsibility of ISO/TC154 standards fulfil their responsibilities.
+      - type: recognises
+        date_effective: 2004-09-10
+        message: ISO/TC154 recognized and appreciates the update provided by OASIS during the March 2006 forum. TC154 acknowledges that OASIS has committed to update their standards contributions, following a schedule consistent with the needs of OASIS, the originating OASIS TC, ISO/TC154 and the broader user community
+      - type: resolves
+        date_effective: 2004-09-10
+        message: ISO/TC154 resolves to request that OASIS continue to keep this TC abreast of submission plans to include projected timelines, for updates to all contributed standards.
+      - type: resolves
+        date_effective: 2004-09-10
+        message: ISO/TC154 also resolves that all other contributors of specifications to ISO/TC154 who retain maintenance responsibility do likewise
+
+
+  - dates:
+      - 2006-03-22
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 272
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: |
+        ISO/TC154, noting that the chair and secretary duty of Mr. Vuilleumier ends in December 2006, resolves
+        - to thank Mr. Vuilleumier for agreeing to fulfil these duties for two more years, up to end of 2008 
+        - to request Mr. Vuilleumier to identify which ISO/TC154 P-member could, in due time, take over these duties
+
+
+  - dates:
+      - 2006-03-22
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 273
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: ISO/TC154 resolves to thank Mr. Duane Nickull, UN/CEFACT Vice Chair, and the Canadian team, for having provided very efficiently the hosting for this meeting
+
+
+  - dates:
+      - 2006-03-22
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Next ISO/TC154 meetings"
+    identifier: 274
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: The ISO/TC154 chair shall consult with the participants of the last ISO/TC154 meetings to set the date and place of this 25th meeting, preferably back-to-back to a relevant other meeting of e.g. ISO/TC184, the UN/CEFACT Forum, the e-Business MoU/MG, or of course of a WG or JWG of ISO/TC154
```

Comment:
Here, I'm using `resolves` action type, but I think I'm wrong. Which word should I use in this case?

Source:
![capture14](https://user-images.githubusercontent.com/33627611/71760014-330d5680-2e8d-11ea-9b87-8e383f62eff2.JPG)

Reply:
I think `resolves` is acceptable, can't think of another word.